### PR TITLE
add FareQuota

### PR DESCRIPTION
**Pull request by ue71603**
fixes https://github.com/NeTEx-CEN/NeTEx/issues/452

### CODE REVIEW

**Review by trurlurl — APPROVED** [review]

File: xsd/netex_part_3/part3_fares/netex_fareProduct_version.xsd [hunk 1]
```diff
@@ -1137,6 +1137,11 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Pricing Elements for a FARE PRODUCT PRICE.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
+			<xsd:element name="FareQuota" type="xsd:integer" minOccurs="0">
```
Comment (skinkie) [inline xsd/netex_part_3/part3_fares/netex_fareProduct_version.xsd:1140]:
```suggestion
			<xsd:element name="FareQuota" type="xsd:nonNegativeInteger" minOccurs="0">
```

**Comment by Aurige** [comment]
We need to discuss this: is the FareProductPrice a really good place for a quota ? Also, what does the quota applies to (the full FARE PRODUCT, a specific JOURNEY to which the product gives access to, etc.).
Up to now we have the FareQuotaFactor and you can assign it via a ValidityPartameterAssignment to whatever it concerns.

**Comment by ue71603** [comment]
@Aurige No worries. We can also NOT do it. We had it done in OJP. In OJPFareResponse a quota makes sense (and it is per journey). So If you don't want this PR. We can just not do it.

**Comment by skinkie** [comment]
> We need to discuss this: is the FareProductPrice a really good place for a quota ? Also, what does the quota applies to (the full FARE PRODUCT, a specific JOURNEY to which the product gives access to, etc.). Up to now we have the FareQuotaFactor and you can assign it via a ValidityPartameterAssignment to whatever it concerns.

I wonder if a quota is also something like should appear in SIRI similar to occupancy? Or is that really an out-of-the-box thought?

**Comment by ue71603** [comment]
@skinkie  It came from the OJPFare2Nova example. The quotas are there. If SIRI has a Fare part we certainly can do it there. But I am not sure, where SIRI is considered with Fares. So I went to NeTEx. But  Christophe is not in favor (at least for the simple solution).

**Comment by ue71603** [comment]
Nick: Can be done already with FareQuota --> Price and not this way around. 
(a) FareQuotaFactor
(b) number of units
(c) have a price.

**Comment by trurlurl** [comment]
Removing "needs documentation update" and "needs transmodel update" since closed without merging.

**Pull request closed without merging**